File: EasyReportDemo/Models/Product.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace EasyReportDemo.Models
{
    [DisplayColumn("Name")]
    public class Product {

        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Id { get; set; }

        public string Name { get; set; }


        [ScaffoldColumn(false)]
        public int? SupplierId { get; set; }

        public string EnglishName { get; set; }

        public string QuantityPerUnit { get; set; }

        public decimal? UnitPrice { get; set; }

        public short? UnitsInStock { get; set; }

        public short? UnitsOnOrder { get; set; }

        public short? ReorderLevel { get; set; }

        public bool Discontinued { get; set; }
    }


}

File: WinForms/EqDemoWinFormsNet4VB/Models/NWind/Product.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

<DisplayColumn("Name")>
Public Class Product
    <DatabaseGenerated(DatabaseGeneratedOption.None)>
    <Column("ProductID")>
    Public Property Id As Integer
    <Column("ProductName")>
    Public Property Name As String
    <ScaffoldColumn(False)>
    Public Property SupplierID As Integer?
    <ForeignKey("SupplierID")>
    Public Overridable Property Supplier As Supplier
    Public Property CategoryID As Integer?
    <ForeignKey("CategoryID")>
    Public Overridable Property Category As Category
    Public Property QuantityPerUnit As String
    Public Property UnitPrice As Decimal?
    Public Property UnitsInStock As Short?
    Public Property UnitsOnOrder As Short?
    Public Property ReorderLevel As Short?
    Public Property Discontinued As Boolean
End Class